specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/DelegatedNetwork.json

```diff
@@ -0,0 +1,1037 @@
+        },
+        "operation": {
+          "description": "Operation type: create, get, delete, etc.",
+          "type": "string",
+          "readOnly": true
+        },
+        "description": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Friendly description for the operation,"
+        }
+      }
+    },
+    "OperationProperties": {
+      "description": "Operation properties.",
+      "type": "object",
+      "properties": {}
+    },
+    "ControllerResource": {
+      "description": "Represents an instance of an DNC controller resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "An identifier that represents the DNC controller resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the DNC controller resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The type of the DNC controller  resource.(Microsoft.DelegatedNetwork/controller)"
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the DNC controller resource."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "DelegatedSubnetResource": {
+      "description": "Represents an instance of a DelegatedSubnet resource.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "An identifier that represents the DelegatedSubnet resource."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the DelegatedSubnet resource."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The type of the DelegatedSubnet  resource.(Microsoft.DelegatedNetwork/delegatedSubnet)"
+        },
+        "location": {
+          "type": "string",
+          "description": "Location of the DelegatedSubnet resource."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "DelegatedSubnetParameters": {
+      "description": "DelegatedSubnet Parameters",
+      "type": "object",
+      "properties": {
+        "controllerID": {
+          "type": "string",
+          "description": "Delegated Network Controller ID"
+        }
+      }
+    },
+    "DelegatedControllers": {
+      "description": "An array of Delegated controller resources.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DelegatedController"
+          },
+          "description": "An array of Delegated controller resources."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DelegatedSubnets": {
```

Comment:
For Java SDK automation error, you may consider to rename this or change the operation ids.
Potential suggestion:
`DelegatedSubnets` -> `DelegatedSubnetList`/`DelegatedSubnetItems`
or
`DelegatedSubnet(s)_OPERATION` -> `DelegatedSubnet(s)Service_OPERATION`

Reply:
done